src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+import java.util.List;
+
+public class Calculator {
+
+    public double calculate(List<Integer> operands, List<Character> operators) {
+        double result = operands.get(0);
+        int numOfOperations = operators.size();
+        int times = 0;
+
+        while (times < numOfOperations) {
+            result = fourFundamentalArithmeticOperations(result, operators.get(times), operands.get(++times));
+        }
+
+        return result;
+    }
+
+    public double fourFundamentalArithmeticOperations(double tempResult, char operator, double operand) {
```

Comment:
메서드 이름:
핵데이 자바 컨벤션 2.10에 메서드 이름은 동사나 전치사로 시작할 것을 권장하고 있네요!
동사로 시작하는 다른 이름을 고려해보는게 어떤가요?

```suggestion
    public double calculateExpression(double tempResult, char operator, double operand) {
```